test/unit/api/events.spec.ts

```diff
@@ -23,6 +17,47 @@ describe('Event Tests', () => {
       .get('/api/getRewards')
       .set('Accept', 'application/json')
       .query({ chain });
-    console.log(res.body);
+    expect(res.body.status).to.be.equal('Success');
+    expect(res.body.validators).to.not.be.null;
+  });
+  it('should fetch global statistics from HistoricalValidator and Validator model', async () => {
```

Comment:
please follow the structure using async/await.

Reply:
Noted